### ISSUE TITLE
[Feedbak] user brews list

### ISSUE THREAD

**Issue by PalaNolho** [stolksdorf/homebrewery#240]
Wow, that was fast :) I was not expecting to see that on the next day :+1: 

If I may, I would like to suggest some adjustments if you think they make sense.

- The titles of the brews seem a bit too big compared with the brew home titles. I would suggest setting the h2 headers font size to 15pt (at least on this page, since this may be a shared CSS that affect the homebrews too)
- If it would be possible, I think the brews widgets (lets call it than for now) should have a fixed size so that they all get aligned correctly with or without description
- I think that 3 lines of description should be enough and if bigger, adding the "..." at the end and putting the full description on a tooltip (like you did before) seems a good option. (using text-overflow: ellipsis ?)
- I would put the visitors and the last update before the users. Most of the cases there will be only 1 user, since people don't usually share their editable brews links. Since this is the case, I think it would make more sense to don't se "myself" on the list and show only users that have actually joined edit rights. This would mean that it would be more informative and easy to see when a page is has "shared editors" and when it doesn't.
-- I would say that the max relevant number for the views would have around 6~8 characters (including a "white-space" and a "+" like "999 999+")
-- the max size for the update would be "A few seconds ago" and anything greater than that user numbers like "2 minutes ago", "3 hours ago", "4 days ago", etc...

http://screencast.com/t/8BIep3dmE
http://screencast.com/t/kMNEtSvU

Hope it helps
Cheers!

**Comment by ericscheid** [comment]
Making all brewItems the same height does align them, but it also makes them look kinda weird when they have no description. Clamping the description to 2 lines does look OK

For style changes, this seems to work..

```
.brewItem { min-height: 80px; }

.brewItem h2 { font-size: 15pt }

.brewItem .description {
    max-height: 2.7em;
    text-overflow: ellipsis;
    overflow: hidden;
    display: -webkit-box;
    -webkit-line-clamp: 2;
    -webkit-box-orient: vertical;
    }
```

**Comment by Gazook89** [comment]
An old topic here but was messing around with my own stylesheet this afternoon for the User Page and made this...erm, monstrosity?  I dunno, clearly I have spent too much time looking at the styling of css-tricks.com.  Making the cards flat and square would fit the current aesthetic better.

<img width="1184" alt="image" src="https://user-images.githubusercontent.com/58999374/127597360-d209c601-5ff7-4a66-9829-780bafd53a80.png">

But I did have a few thoughts that maybe aren't terrible(?):
1. Possibility of custom user-created stylesheets for their user pages?  This might be an "easy" way of adding some *meat* to the user account 'experience', which compared to GMBinder is pretty lacking.  Allowing users to add their own personality to their user page is "fun"....you know, like MySpace.
2. The html elements on the user page suffer from lack of class names or ID's.  The previous suggestion would need a better naming convention.  Giving each `.brewItem` an id# that matches the brew name, and the container div around each the Published and Unpublished brews class names would be helpful.
3. I know there has been talk of tagging/grouping, but in the meantime it would seem that the selected game system for a brew could be represented on the `.brewItem`s.   I envision either small color coded labels below the current `.info` div listing the system and other tags, or using the game system to determine the entirety of each `.brewItem` card's background color.
4. Adding another text field in the metadata editor that takes an image URL, which is then displayed in the background of the corresponding `.brewItem` card.   It could be done with the image on the right side of the card, and a nice fade going left.  However, if people make many many brews and have an image for every one, i can image running into issues (partial page rendering?).

**Comment by ericscheid** [comment]
1. also pertinent when we make it easy for users to share their folders of brews. Oh .. different style sheets per folder .. gosh!
2. especially useful once we have dynamic filtering on-page .. can't rely on `:nth-child(n)` there. Could also use the shareId as the unique id to allow non-fragile renaming of brews.
3. I enisage tags similar to GMB, but for the standard tag groups have an additional tint .. e.g. all `system` tags are green, all `type` tags are blue, all user-custom tags are white.
4. this could also be done via custom css if each brewItem had a unique id (see your point # 2 above) .. although if we also want to display the custom image when the brewItem is displayed in a site-wide search we will need the easier method of storing that meta-data in the brew-meta (a la GMB).
5. agree. the common action is to either edit-if-author or view-share-if-visitor. Make that the easiest

![image](https://user-images.githubusercontent.com/2605467/127612970-16435c9d-84c4-424a-b9e2-462a1c4e70ba.png)

**Comment by ericscheid** [comment]
Another loose end is the entire section "Unpublished Brews" should not be shown at all if not looking at ones own UserPage. There will never be anything listed there.

![image](https://user-images.githubusercontent.com/2605467/128631828-d0dd4918-9611-4f8c-853d-d0d7bd8712ba.png)

**Comment by Gazook89** [comment]
> Another loose end is the entire section "Unpublished Brews" should not be shown at all if not looking at ones own UserPage. There will never be anything listed there.

I think this is fixed.

<img width="955" alt="image" src="https://user-images.githubusercontent.com/58999374/133278088-de672804-a825-478c-b56a-d192fe303710.png">

**Comment by calculuschild** [comment]
@Gazook89  Yep that is fixed. In fact, I think most of the original post on this issue has now been addressed. Perhaps we can close this and if there are any suggestions from those later discussions that are still relevant we can put them into a new clean issue.
